requirements.txt

```diff
@@ -1,3 +1,4 @@
 requests
-django>=3.0.5
+django>=3.1
+django-jsonfield
```

Comment:
I think django-jsonfield is included into django>=3.1 (which is the latest release) ... if we want to include support for 3.0.5 this might help - but I'm not sure even in that case.